emails/views.py

```diff
@@ -123,7 +123,8 @@ def _index_POST(request):
         )
         return JsonResponse({
             'id': relay_address.id,
-            'address': address_string
+            'address': address_string,
+            'domain': relay_addres.domain_value
```

Comment:
If we want to add the `domain` property to the response that's good, but this doesn't fix the issue, because the `address_string` is still `'%s@%s` where the 2nd string substitution is based on the `relay_from_domain`. So the add-on will need to:

1. get the localpart from the `address` value by splitting it at the `@` character
2. append the domain part from the `domain` value

AND we will need to coordinate the release of the add-on with the additional parsing code with the release of the server code.

A better fix that de-couples the code-bases, but still extends the response in a way the add-on can use later:
```
diff --git a/emails/models.py b/emails/models.py
index 621589b..fb62ff0 100644
--- a/emails/models.py
+++ b/emails/models.py
@@ -272,6 +272,10 @@ class RelayAddress(models.Model):
     def domain_value(self):
         return DOMAINS.get(self.get_domain_display())
 
+    @property
+    def full_address(self):
+        return '%s@%s' % (self.address, self.domain_value)
+
 
 class DeletedAddress(models.Model):
     address_hash = models.CharField(max_length=64, db_index=True)
diff --git a/emails/views.py b/emails/views.py
index ec587a8..790c529 100644
--- a/emails/views.py
+++ b/emails/views.py
@@ -115,12 +115,10 @@ def _index_POST(request):
             return redirect('profile')
 
     if settings.SITE_ORIGIN not in request.headers.get('Origin', ''):
-        address_string = '%s@%s' % (
-            relay_address.address, relay_from_domain(request)['RELAY_DOMAIN']
-        )
         return JsonResponse({
             'id': relay_address.id,
-            'address': address_string
+            'address': relay_address.full_address,
+            'domain': relay_address.domain_value
         }, status=201)
 
     return redirect('profile')
```